Realm/RLMSyncUser.h

```diff
@@ -269,13 +276,47 @@ NS_SWIFT_UNAVAILABLE("Use the full version of this API.");
 - (void)revokePermission:(RLMSyncPermission *)permission callback:(RLMPermissionStatusBlock)callback;
 
 /**
- Returns an instance of the Management Realm owned by the user.
+ Create a permission offer for a Realm.
+
+ A permission offer is used to grant a specific permission to a Realm you are allowed to manage
```

Comment:
I found it hard to parse this sentence. Maybe something like "A permission offer is used to grant access to a Realm you manage to another user."

"You" is also inaccurate here, but it is more concise than "the current user" and people will probably understand what it means?

Reply:
Thanks for the suggestion; it is a significant improvement. Also, I was a little worried that using "you" might be too informal or imprecise, and given your feedback I think it's probably worthwhile to state the actor more accurately.